tests/doc_scan_service/session/create/session.specification.builder.test.js

```diff
@@ -159,4 +175,46 @@ describe('SessionSpecificationBuilder', () => {
 
     expect(JSON.stringify(sessionSpec)).toBe(expectedJson);
   });
+
+  it('should build SessionSpecification without identityProfile', () => {
+    const sessionSpec = new SessionSpecificationBuilder()
+      .build();
+
+    const expectedJson = JSON.stringify({
+      requested_checks: [],
+      requested_tasks: [],
+      required_documents: [],
+    });
+
+    expect(JSON.stringify(sessionSpec)).toBe(expectedJson);
+  });
```

Comment:
I see the intention, but I don't think that this is valuable to add.
As a matter of fact, previous test are already asserting that when you do not call `.withIdentityProfile` (soon to be renamed), then the payload does not include it.
So I think we can remove this test.